docs/source/en/tasks/translation.md

```diff
@@ -348,7 +348,9 @@ The simplest way to try out your finetuned model for inference is to use it in a
 ```py
 >>> from transformers import pipeline
 
->>> translator = pipeline("translation", model="my_awesome_opus_books_model")
+# Change `xx` to the language of the input and `yy` to the language of the desired output.
+# Examples: "en" for English, "fr" for French, "de" for German, "es" for Spanish, "zh" for Chinese, etc; translation_en_to_fr translates English to French
```

Comment:
could you link to the full list of languages?

Reply:
Sure @ArthurZucker. Do you mean this list - https://huggingface.co/languages ?